pages/sdk/signers/passkeys.mdx

```diff
@@ -98,44 +100,68 @@ In this guide, you will learn how to create a Passkey signer that can be added a
     throw Error('Passkey creation failed: No credential was returned.')
   }
 
-  const passkey = await extractPasskeyData(passkeyCredential)
+  const passkeySigner = await Safe.createPasskeySigner(passkeyCredential)
   ```
 
   {/* <!-- vale on --> */}
 
-  At this point, it's critical to securely store the information in the `passkey` object in a persistent service. Losing access to this data will result in the user being unable to access their passkey and, therefore, their Safe Smart Account.
+  At this point, it's critical to securely store the information in the `passkeySigner` object in a persistent service. Losing access to this data will result in the user being unable to access their passkey and, therefore, their Safe Smart Account.
 
   ### Get the provider and signer
 
-  Once the passkey is created, you can get the `provider` and `signer`, which is the externally-owned account of the user that was derived from its credentials.
+  Once the passkey is created, you need the `provider` and `signer` props that the Safe\{Core\} SDK require to instantiate the kits.
 
+  [How to initialize the Protocol Kit](https://docs.safe.global/reference-sdk-protocol-kit/initialization/init)
+
+  The `provider` (eip1193 compatible or rpc) can be obtained by using `viem` in several ways:
   {/* <!-- vale off --> */}
 
-  <Tabs items={['viem']}>
+  <Tabs items={['eip1193', 'rpcUrl']}>
     <Tabs.Tab>
-      You can instantiate the provider using `viem` and the following imports:
-
-      ```typescript
-      import { createWalletClient, http } from 'viem'
-      import { sepolia } from 'viem/chains'
-      ```
+    ```typescript
+    import { createWalletClient, http } from 'viem'
+    import { sepolia } from 'viem/chains'
 
-      ```typescript
-      const provider = createWalletClient({
-        chain: sepolia,
-        transport: http('https://rpc.ankr.com/eth_sepolia')
-      })
+    const walletClient = createWalletClient({
+      chain: sepolia,
+      transport: http('https://rpc.ankr.com/eth_sepolia')
+    })
 
-      const signer = passkey
+    const provider = { request: walletClient.request }
+    const signer = passkeySigner
       ```
+          </Tabs.Tab>
+    <Tabs.Tab>
+    ```typescript
+    import { sepolia } from 'viem/chains'
+
+    const provider = sepolia.rpcUrls.default.http[0]
+    const signer = passkeySigner
+    ```
```

Comment:
I think this is out of scope for this PR.
The used `provider` is just an example to provide to the kits, all the different ways to create it are covered in the reference here: https://docs.safe.global/reference-sdk-protocol-kit/initialization/init#provider

In fact, I think the `provider` could also be reemoved as this is just about the generating the `signer`. This also applies to all the Safe Signers guides, so I think it's out of scope for this PR.

Reply:
What is ooo? the two examples or the next section "Instantiate the SDK"?

Reply:
The changes inside ```<Tabs items={['eip1193', 'rpcUrl']}>```

Reply:
done